cli/exchange/business.go

```diff
@@ -86,6 +86,17 @@ func BusinessAddPolicy(org string, credToUse string, policy string, jsonFilePath
 		cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("Incorrect deployment policy format in file %s: %v", jsonFilePath, err))
 	}
 
+	cons := len(policyFile.Constraints)
+	if noConstraints {
+		if cons > 0 {
+			cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("--no-constraints flag is specified but the given deployment policy contains %v constraints.", cons))
+		}
+	} else {
+		if cons == 0 {
+			cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("You must use --no-constraints flag because the the given deployment policy does not contain any constraints."))
```

Comment:
Please change the error message to: "The deployment policy has no constraints which might result in the service being deployed to all nodes. Please specify --no-constraints to confirm that this is acceptable."

Reply:
Done